src/Identity.Dapper.MySQL/TypeHandlers/MySqlDateTimeOffsetHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Dapper;
+using System;
+using System.Data;
+
+namespace Identity.Dapper.MySQL.TypeHandlers
+{
+    public class MySqlDateTimeOffsetHandler : SqlMapper.TypeHandler<DateTimeOffset?>
+    {
+        // assume UTC in and out
+
+        public override DateTimeOffset? Parse(object value)
+        {
+            if (value == null || value == DBNull.Value) return null;
+
+            var result = DateTime.SpecifyKind((DateTime)value, DateTimeKind.Utc);
```

Comment:
What about people that is already using Identity.Dapper and have data on their DB? Assuming that his data is UTC could lead to problems, no?

Reply:
The way I look at it is - there are 2 possible scenarios for existing ```Identity.Dapper``` users:
1. Users who haven't done anything and are using Identity.Dapper as it is. In this case they are getting wrong lockout periods right now. Keep the test, but comment out the change and you should see this.
2. Users who have done some arrangements to handle the situation. What they might (or might not)  need to change depends on what have they done. But in any case it is outside of Identity.Dapper.  This a fix is for ```Identity.Dapper``` to correctly handle ```DateTimeOffset``` values in ```MySQL```. It doesn't make sense to keep it broken/incomplete, just because someone might (or might not) have implemented a workaround in their code. One would hope that if someone found the problem and implemented a workaround, they would have also reported it to ```Identity.Dapper```...right?

I suppose another option is to provide the type handler, but leave it up to the user to call ```SqlMapper.AddTypeHandler```.

Reply:
Makes sense. `DateTimeOffset` is used just at `LockoutEndDate`.

Reply:
Makes sense. DateTimeOffset is used just at LockoutEndDate.